subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/samples/dependencymanagement/SamplesDeclaringDependenciesIntegrationTest.groovy

```diff
@@ -26,12 +27,15 @@ class SamplesDeclaringDependenciesIntegrationTest extends AbstractIntegrationSpe
 
     private static final String COPY_LIBS_TASK_NAME = 'copyLibs'
 
+    File repositoryMirrorScript = RepoScriptBlockUtil.createMirrorInitScript()
+
     @Rule
     Sample sample = new Sample(testDirectoryProvider)
 
     @UsesSample("userguide/dependencyManagement/declaringDependencies/concreteVersion")
     def "can use declare and resolve dependency with concrete version"() {
         executer.inDirectory(sample.dir)
+        executer.usingInitScript(repositoryMirrorScript)
```

Comment:
We can probably avoid a couple lines of code if we'd put that into a `setup` block.

Reply:
Not all tests are needing it, but since it should be a no-op then, that's probably cleaner